Classes/Issues/Labeled/IssueLabeledSectionController.swift

```diff
@@ -28,8 +32,13 @@ final class IssueLabeledSectionController: ListGenericSectionController<IssueLab
             let object = self.object
             else { fatalError("Missing collection context, cell incorrect type, or object missing") }
         cell.configure(object)
-        cell.delegate = viewController
+        cell.delegate = self
         return cell
     }
-
+    
+    func didTap(cell: MarkdownStyledTextView, attribute: DetectedMarkdownAttribute) {
+        guard case .label(let label) = attribute else { return }
+        tapDelegate?.didTapIssueLabel(owner: label.owner, repo: label.repo, label: label.label)
+    }
```

Comment:
Does this break showing the profile when tapping on the username in the labeled cell?

Reply:
Yes it does- updated to: 
```swift 
 if case .label(let label) = attribute {
      tapDelegate?.didTapIssueLabel(owner: label.owner, repo: label.repo, label: label.label)
 } else {
      viewController?.handle(attribute: attribute)
 }
```